lib/profile/config.rb

```diff
@@ -34,6 +34,12 @@ def hunter_command
             config.hunter_command ||
             File.join(ENV.fetch('flight_ROOT', '/opt/flight/'), 'bin/flight hunter')
         command.split(' ')
+        if !File.exists?(File.join(command[0], command[1]))
```

Comment:
We ought to be using `!File::file?` here. Not only is `File::exists?` deprecated in favour of `File::exist?`, but `File::exist?` returns `true` if the given pathname is a file _or_ a directory. We wouldn't want a `true` return value for the latter.